### ISSUE TITLE
Create an inline documentation (similar to JSDoc)

### ISSUE THREAD

**Issue by siv2r** [siv2r/kidney-exchange#26]

**Comment by araiz-1999** [comment]
Would like to work on it, is it available ?

**Comment by M2kishore** [comment]
This picture in README.md is not compatible with dark mode. The words need more vibrant colors like green, red, yellow, etc to make them visible in both modes.

![image](https://user-images.githubusercontent.com/60045529/112708106-000e1300-8ed6-11eb-82d2-34240588e327.png)

**Comment by siv2r** [comment]
Sure!! Feel free to edit it and send a PR :)

**Comment by M2kishore** [comment]
May I know the software or link you used?
Sharing the .xml will be very helpful.
> Sure!! Feel free to edit it and send a PR :)

**Comment by siv2r** [comment]
@M2kishore The change was made by one of the contributors so, I am not sure about the details.
You can take a look at the commit 1d296ce to get more info.

**Comment by nityaak5** [comment]
I would like to work on the issue. Is it still available?

**Comment by siv2r** [comment]
@nityaak5 Sure! Go ahead :+1: I updated the description of this issue, which will give you a rough idea to start. You can also mention what you would like to work on.

Do let me know if you have any doubts or need any help :)

**Comment by darpankhanna** [comment]
@siv2r I want to work on this issue

**Comment by siv2r** [comment]
@darpankhanna I so nice to see contributors wanting to work on documentation issue :)

Please go ahead. Feel free to ask ANY doubts on the discord grp

**Comment by priyanshuone6** [comment]
Few things to change in the README
- Add `License` before the `Contributors`
- Add `Get in touch`, remove the slack group point from `Contributing Guidelines` and add it here (take hints from [here](https://github.com/pints-team/pints#get-in-touch))
- Put the logo before the Kidney Exchange Program 
- Add few badges (be creative 😉)
- Move `CreatePr.md`, `CONTRIBUTING.md` in the docs folder and update their links in the README

**Comment by subhangi2731** [comment]
please assign me this

**Comment by darpankhanna** [comment]
@subhangi2731 i am already assigned with this issue :)

**Comment by siv2r** [comment]
@subhangi2731 sure go ahead!!
feel free to ask your doubts on the discord group :)
Also, @darpankhanna is working on creating an issue template so, you can try working on other things so your works does not clash

**Comment by siv2r** [comment]
> Few things to change in the README
> 
> * Add `License` before the `Contributors`
> * Add `Get in touch`, remove the slack group point from `Contributing Guidelines` and add it here (take hints from [here](https://github.com/pints-team/pints#get-in-touch))
> * Replace the logo3.png in the images folder with [this image](https://drive.google.com/file/d/1TLILWNgSm0KL4GpFgQjK-yFKFjLfKHlR/view?usp=sharing) (as it is cropped and centred)
> * Put the logo before the Kidney Exchange Program and change img width and height to 40
> * Add few badges (be creative wink)
> * Move `CreatePr.md`, `CONTRIBUTING.md` in the docs folder and update their links in the README
> * Add project screenshots in the `README`

added these ideas to the description :)

**Comment by siv2r** [comment]
@subhangi2731, are u working on this issue? If you need any help or have any doubts, you can ask in the discord groups.

**Comment by subhangi2731** [comment]
yes @siv2r I am working on this I have already worked I am sending a pr now

**Comment by subhangi2731** [comment]
please assign me the level tag in this issue @siv2r

**Comment by priyanshuone6** [comment]
@subhangi2731 I think that the level tag is only checked on the pull requests and not on issues

**Comment by siv2r** [comment]
yep, I will assign the level tags to your PR before merging them.

**Comment by Astitva7717** [comment]
Can I do this?

**Comment by siv2r** [comment]
@Astitva7717 Sure, go ahead!
Some of it has been implemented. So try the unimplemented ones or if you have any new ones.

**Comment by riasenpai** [comment]
@Astitva7717 any updates?
